FirebaseEmailAuthUI/FUIEmailAuth.h

```diff
@@ -0,0 +1,89 @@
+//
+//  Copyright (c) 2018 Email Inc.
```

Comment:
@Yue-Wang-Google Please update the organization in your Xcode config

Reply:
I'm sorry. Pull request #544 is created to fix this.

Reply:
Thanks, and no problem 😃